lib/events/api.go

```diff
@@ -28,6 +28,8 @@ import (
 const (
 	// EventType is event type/kind
 	EventType = "event"
+	// EventUID is a unique event identifier
+	EventUID = "uid"
```

Comment:
why not just `ID` ?

Reply:
`id` is already being used for another field (`EventCursor`) which as far as I understand is not persisted but gets assigned on demand when iterating over events: https://github.com/gravitational/teleport/blob/master/lib/events/api.go#L48

Reply:
@klizhentas Anyway, I've renamed `EventUID` to `EventID` and `GetUID()` to `GetID()`, it's indeed a bit cleaner this way. LMK if this looks good to you now.